doc/concepts.rst

```diff
@@ -657,7 +657,7 @@ will match files that exist in the source tree as well as buildable targets, so
 for instance you can depend on ``*.cmi``.
 
 Dune supports globbing files in a single directory via ``(glob_files
-...)`` and in all sub-directories recursively via ``(glob_files_rec
+...)`` and starting with Dune 3.0, in all sub-directories recursively via ``(glob_files_rec
```

Comment:
```suggestion
...)`` and, starting with Dune 3.0, in all sub-directories recursively via ``(glob_files_rec
```

Reply:
Thanks @christinerose , will update asap.

A thing we don't do very uniformly in the manual is to keep a "changelog" of versions and introduced features, so that's something maybe to think about.